roles/ceph-mon/meta/main.yml

```diff
@@ -11,4 +11,4 @@ galaxy_info:
   categories:
     - system
 dependencies:
-  - { role: ceph-common, when: not mon_containerized_deployment }
```

Comment:
maybe it's part of your previous patches but why do we need to include `ceph-common` role for docker?